lib/opal/parser/with_c_lexer.rb

```diff
@@ -1,10 +1,11 @@
 # frozen_string_literal: true
 
-begin
-  require 'c_lexer'
-rescue LoadError
-  $stderr.puts 'Failed to load WithCLexer, using pure Ruby lexer'
-end
+# There's no compatible c_lexer for parser 3.0.0.0 at this point...
+# begin
+#   require 'c_lexer'
+# rescue LoadError
+#   $stderr.puts 'Failed to load WithCLexer, using pure Ruby lexer'
+# end
```

Comment:
I'm onboard with the change, and actually had a similar change hanging around locally…
I'd like to avoid shipping the commented code tho, can you wrap the output with `if $DEBUG`?

```diff
-  $stderr.puts 'Failed to load WithCLexer, using pure Ruby lexer'
+  if $DEBUG
+    $stderr.puts 'Failed to load WithCLexer, using pure Ruby lexer'
+  end
```